tensorflow/core/platform/default/logging.cc

```diff
@@ -109,14 +109,29 @@ int64 MinLogLevel() {
   }
 }
 
+int64 MinLogLevelFromEnv() {
+  const char* tf_env_var_val = getenv("TF_CPP_MIN_LOG_LEVEL");
+  return LogLevelStrToInt(tf_env_var_val);
+}
+
+int64 MinVLogLevelFromEnv() {
+  const char* tf_env_var_val = getenv("TF_CPP_MIN_VLOG_LEVEL");
```

Comment:
@vrv, Yeah agree, I noticed this implementation I thought it was by design.

std::stoi doesn't look safe since it throws exception when input is wrong, see http://en.cppreference.com/w/cpp/string/basic_string/stol.

Can we use atol instead, it returns 0 on error, looks like what we want. 

Thoughts?